assets/javascripts/discourse/components/chat-selection-manager.js

```diff
@@ -0,0 +1,117 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { clipboardCopyAsync } from "discourse/lib/utilities";
+import { getOwner } from "discourse-common/lib/get-owner";
+import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { isTesting } from "discourse-common/config/environment";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { schedule } from "@ember/runloop";
+import { inject as service } from "@ember/service";
+import getURL from "discourse-common/lib/get-url";
+
+export default Component.extend({
+  tagName: "",
+  router: service(),
+  chatChannel: null,
+  selectedMessageIds: null,
+  showChatQuoteSuccess: false,
+  cancelSelecting: null,
+
+  @discourseComputed("selectedMessageIds")
```

Comment:
you can compute on length directly, and we also have a gt() helper

Reply:
Nice will change

Reply:
Tried to use `gt` and `discourseComputed` on the length, and neither updated when the selected messages changed in chat live pane. Ended up with the normal ember `@computed` property:

```javascript
  @computed("selectedMessageIds.length")
  get anyMessagesSelected() {
    return this.selectedMessageIds.length > 0;
  }
```